documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
+
+If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier you have for this organisation.
     
-    If you only record data in a spreadsheet, and don't assign organisations an ID, you could use a spreadsheet formula to turn the organisation name into an identifier (e.g. removing spaces and lowercasing the name). The support team can provide guidance on this. 
+If you use a database that records details of organisations in a separate lookup table, this may provide an identifier you can use. 
     
-    If there is a chance that your organisation identifiers might overlap with grant identifiers, just add 'ORG' into the identifier string (e.g. '360G-xyztrust-ORG123')
+If you only record data in a spreadsheet, and don't assign organisations an ID, you could use a spreadsheet formula to turn the organisation name into an identifier (e.g. removing spaces and lowercasing the name). The support team can provide guidance on this. 
     
-### Company Numbers and Charity Numbers
+If there is a chance that your organisation identifiers might overlap with grant identifiers, just add 'ORG' into the identifier string (e.g. '360G-xyztrust-ORG123')
 
-Sometimes you may have recorded both the company number, and charity number, of an organisation in your data. Because having both of these is important, 360Giving also includes two extra separate fields for ```Company Number``` and ```Charity Number```. If you have these details, you should fill them in, in addition to providing the unique organisation identifier using the method above. 
+```eval_rst
+
+.. admonition:: Special 360 giving fields for charity and company number
 
+   Because 'Company Number' and 'Charity Number' are so important for analysing grantmaking in the UK, the 360 Giving Standard includes additional special fields these on their own (without the prefixes), to help users of the data. 
 
-## Codes
+   If you have these details, you should fill them in, **in addition to** providing the unique organisation identifier using the method above. 
+```
+
+### Collecting organisation identifiers
+
+Make sure that when you collect charity or company numbers, you clearly identifier in your forms of systems what kind of number they are. 
+
+
+
+```eval_rst
+.. _codes:
+```
+
+## Classifications
 
 [Contact the support team](http://www.threesixtygiving.org/contact/) for details of prefixes to use when publishing classification codes for your grants.
+
+## A step further: using identifiers to join up information
+
+We use simple text identifiers in 360Giving. As well as using these identifiers in your published data, you could also tell your grantees their prefixed identifier, and encourage them to include it in any documents they publish about their funded projects, or to include it as a 'machine tag' (or hashtag) when uploading photos and videos to social media sites. 
```

Comment:
While this is cool and good to know, I feel this is going to confuse our publishers even more. I don't think they even in the stage of doing this with their grantees...

Reply:
I suggest we remove this recommendation. Its unlikely to happen and we already to know the reporting burden on grantees is high. It may also put some organistions off from publishing as they may think they need to ask their grantees to do this or seek their permission in some way - we're asked about this a lot and there's a risk we send mixed messages here. The burden of publishing to 360Giving should lie solely with the grantmaker.